packages/fxa-auth-server/test/remote/oauth_tests.js

```diff
@@ -312,4 +312,52 @@ describe('/oauth/ routes', function() {
     const clientRotatedJWT = decodeJWT(clientRotatedRes.access_token);
     assert.notEqual(tokenJWT.claims.sub, clientRotatedJWT.claims.sub);
   });
+
+  it('successfully revokes access tokens, and refresh tokens', async () => {
+    let res = await client.createAuthorizationCode({
+      client_id: PUBLIC_CLIENT_ID,
+      state: 'abc',
+      code_challenge: MOCK_CODE_CHALLENGE,
+      code_challenge_method: 'S256',
+      scope: 'profile openid',
+      access_type: 'offline',
+    });
+    assert.ok(res.code);
+
+    res = await client.grantOAuthTokens({
+      client_id: PUBLIC_CLIENT_ID,
+      code: res.code,
+      code_verifier: MOCK_CODE_VERIFIER,
+    });
+    assert.ok(res.access_token);
+    assert.ok(res.refresh_token);
+
+    await client.revokeOAuthToken({
```

Comment:
You should be able to call [`/introspect`](https://github.com/mozilla/fxa/blob/86daadeb8bba29f994291f7829f9d59952166701/packages/fxa-auth-server/fxa-oauth-server/lib/routes/introspect.js#L22) with this access token to see if has actually been invalidated.